ui/app/models/pki/pki-role-engine.js

```diff
@@ -6,25 +6,63 @@ import { withModelValidations } from 'vault/decorators/model-validations';
 import fieldToAttrs from 'vault/utils/field-to-attrs';
 
 const validations = {
-  name: [
-    { type: 'presence', message: 'Name is required.' },
-    {
-      type: 'containsWhiteSpace',
-      message: 'Name cannot contain whitespace.',
-    },
-  ],
+  name: [{ type: 'presence', message: 'Name is required.' }],
 };
 
 @withModelValidations(validations)
 export default class PkiRoleEngineModel extends Model {
   @attr('string', { readOnly: true }) backend;
   @attr('string', {
     label: 'Role name',
-    fieldValue: 'id',
+    fieldValue: 'name',
     readOnly: true,
   })
   name;
 
+  // ARG TODO: will have to handle in serializer to send in the correct params
+  @attr({
+    label: 'Not valid after',
+    subText:
+      'The time after which this certificate will no longer be valid. This can be a TTL (a range of time from now) or a specific date.',
+    editType: 'yield',
```

Comment:
here is the new custom editType. Right now, I am not sending this data correctly on the network request, but I'll handle that in a future PR. This PR focused on getting the default fields showing on the form per the designs and firing off a createRecord.